styles/scss/geoChart.scss

```diff
@@ -13,42 +14,72 @@
         width: 100%;
     }
     .geo-legend {
-        padding-bottom: 5px;
-        float: right;
         display: flex;
+        flex-flow: row wrap;
+        align-items: center;
+        justify-content: flex-end;
+        width: 100%;
+        > div {
+            margin: 5px 0;
+        }
+        .viz-static-legend-wrap {
+            margin-right: 20px;
+            flex: 60%;
+            align-items: flex-end;
+            justify-content: flex-end;
+        }
         .pushpin-size-legend {
             display: flex;
             margin-left: 10px;
+            font-family: "Avenir", "Helvetica Neue", Arial, sans-serif;
```

Comment:
use `$gd-font-primary` instead of static string

Reply:
fixed